Updated Starlette/FastAPI setup documentation (#5472)
getsentry/sentry-docs: src/platforms/python/guides/starlette/index.mdx

The Starlette integration adds support for the [Starlette Framework](https://www.starlette.io/).

> Sentry SDK will auto enabled support for Starlette if you have the `starlette` Python package installed in your project. There is no configuration options you need to add when initializing the Sentry SDK, everything works out of the box.

Reviewer: ```suggestion
The Sentry SDK automatically enables support for Starlette if you have the `starlette` Python package installed in your project. There are no configuration options you need to add when initializing the Sentry SDK, as everything works out of the box.
```
